pkg/skaffold/tag/git_commit_test.go

```diff
@@ -85,12 +88,29 @@ func TestGitCommit_GenerateTag(t *testing.T) {
 			},
 		},
 		{
-			description:            "dirty worktree with tag containing a slash",
+			description:            "clean worktree with a branch",
```

Comment:
can you add a test case for branch name with a slash like `branch/foo`? The tag should then be `branch_foo`.

Reply:
great idea!